runtime/integration-tests/src/rocc/pool_starts.rs

```diff
@@ -9,7 +9,7 @@
 // but WITHOUT ANY WARRANTY; without even the implied warranty of
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
-use crate::chain::centrifuge::PARA_ID;
+use crate::chain::centrifuge::{Runtime as CentrifugeRuntime, PARA_ID};
```

Comment:
Really not a big fan of renaming imports and it would read super nice to refer to these items as `centrifuge::Runtime` and `centrifuge::PARA_ID`.

Reply:
I will rename it in the afterwards PR. Wdyt? Named it like this as I thought at the beginning I will also need the relay runtime but I think this won't be necessary.

Reply:
No biggy, we can always iterate on this later on, let's get the 🥩 on the table first yes :)